xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
    </para>
   </sect1>
   <xi:include href="hardening_pam_stack.xml"/>
+  <sect1 xml:id="sec-sec-prot-restrict-root">
+   <title>Restricting &rootuser; logins</title>
+
+   <para>
+    By default the &rootuser; user is assigned a password
+    and can login on any local terminal, in a graphical session or via SSH.
+    For security sensitive systems &rootuser; logins should be
+    restricted to as few methods as possible. Shared usage of the root account
```

Comment:
> "For security sensitive systems"... that sounds like there are systems which don't need security. I think, this gives the wrong impression. We should tell our customers that security is _always_ important.

I see what you're getting at. But there is a difference between, say, a workstation in a restricted office, a server in a data center and a laptop used in the field. It could also raise the question why these hardenings need to be applied manually then and aren't already setup by default ;-). In this light I think it is legit to notice that these hardenings are not for everybody but for a specific target audience that worry especially about security or have especially sensitive setups. But I'm still okay to drop this part.

> Another issue is, I'm not sure what you mean by "methods". Accounts? As such, I'd suggest to rephrase it like:

Actually, no. The methods refer to the "local terminal", "graphical session" and "SSH", different ways to login as root on a system.

Reply:
Ahh, I see what you mean. 🙂 Is that a common term? I heard it before. It could make sense to little bit elaborate on that. It doesn't need much, but maybe an additional sentence would be fine. 😉

Oh, then I misread your sentence, sorry. Well, in that case would it make sense to add an additional hint? Something like this:

    logins should be restricted to as few methods (like local terminal, SSH, graphical sessions, etc.) 
    as possible.

Reply:
I think that would be strangely redundant. I try a bit of a different approach to make it more clear.

You mean if "login methods" is a common term? Well Google knows a couple of million of occurrences ;-). Apart from that I don't think the term "method" is uncommon in technical contexts.

Reply:
Hmn, yes. Ok, I'll leave that to Liam, maybe he has some idea.

Reply:
I think that "login method" is a valid phrase, but ambiguous. I would just suggest slightly clarifying the list of examples:

> logins should be restricted to as few methods (such as on the local console, via SSH, graphical sessions, etc.) 

I think with the examples, it's clear.